examples/typescript/node.api.js

```diff
@@ -0,0 +1,12 @@
+// Paths Aliases defined through tsconfig.json
+const typescriptWebpackPaths = require('./webpack.config.js')
+
+export default () => ({
+  webpack: (config) => {
+    // Add TypeScript Path Mappings (from tsconfig via webpack.config.js)
+    // to react-statics alias resolution
+    config.resolve.alias = typescriptWebpackPaths.resolve.alias
```

Comment:
Personally a fan of Object.assign. @tannerlinsley any preference?

Hold on any changes till author confirms preference

Reply:
Yes do an assignment or extension. You don't want to clobber other plugin or user changes

Reply:
Dot Notation is dangerous too because it relies on the integrity of the object structure, Object.assign is a little better